plugins/view-transitions/includes/settings.php

```diff
@@ -235,17 +252,29 @@ function plvt_apply_settings_to_theme_support(): void {
  * @access private
  */
 function plvt_add_setting_ui(): void {
+	global $plvt_has_theme_support_with_args;
+
 	add_settings_section(
 		'plvt_view_transitions',
 		_x( 'View Transitions', 'Settings section', 'view-transitions' ),
 		static function (): void {
+			global $plvt_has_theme_support_with_args;
 			?>
 			<p class="description">
 				<?php esc_html_e( 'This section allows you to control how view transitions are used to enhance the navigation user experience.', 'view-transitions' ); ?>
 				<br>
 				<?php esc_html_e( 'To reset any of the selector text inputs, clear the field and save the changes.', 'view-transitions' ); ?>
 			</p>
 			<?php
+			if ( $plvt_has_theme_support_with_args ) {
+				wp_admin_notice(
+					__( 'Your theme already supports view transitions with its own adapted configuration. The settings below will override those.', 'view-transitions' ),
+					array(
+						'type'               => 'warning',
```

Comment:
Since we're simply informing the user about the setting rather than alerting them to a problem, would it make more sense to use an `info` notice instead of a `warning`?

Reply:
Fair point, will update.